packages/react/src/composite/ScrollPane/src/components/ScrollPaneArrow/ScrollPaneArrow.tsx

```
@@ -0,0 +1,105 @@
+  }
+};
+
+  return (
+    <div
+      className={[
+        scrollPane.classnames.arrow,
+        scrollPane.classnames.arrowVariants[variant],
+      ].join(" ")}
+      onClick={handleActivation}
+      onKeyDown={handleKeyDown}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)